packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -6,7 +6,11 @@
  */
```

Comment:
The product-detail test passes, but I'm seeing warnings like "Unrecognized customer type found in storage:". Do you see that in your tests too? Should I resolve these warnings somehow?

![iTerm2 2023-03-16 at 15 23 46](https://user-images.githubusercontent.com/847300/225766738-9b594ac7-c7b8-46fe-ae14-23bd99617d51.png)